api/config/swagger.py

```diff
@@ -487,8 +498,30 @@ def schema_view_swagger(request):
                     ],
                     description = 'Update an existing answer'
                 ),
+                'Delete answer': coreapi.Link(
```

Comment:
Rebase and test the documentation,
If this is added to the line specified in develop it will be under answers instead of questions

Reply:
@Ogutu-Brian I have fixed it